datahub-web-react/src/app/entity/shared/containers/profile/utils.ts

```diff
@@ -202,3 +202,22 @@ export function getOnboardingStepIdsForEntityType(entityType: EntityType): strin
             return [];
     }
 }
+
+function sortTabsWithDefaultTabId(tabs: EntityTab[], defaultTabId: string) {
+    return tabs.sort((tabA, tabB) => {
+        if (tabA.id === defaultTabId) return -1;
+        if (tabB.id === defaultTabId) return 1;
+        return 0;
+    });
+}
+
+export function sortEntityProfileTabs(appConfig: AppConfig, entityType: EntityType, tabs: EntityTab[]) {
+    const sortedTabs = [...tabs];
+
```

Comment:
Nice I see we are only support domain reordering for now.. I think this makes sense!

Reply:
yup!